### ISSUE TITLE
Error: no such option: --file

### ISSUE THREAD

**Issue by HenrikBach1** [Azure/iotedgehubdev#181]
Hi,

Following your `Quickstart with IoT Edge Dev Container`, item 5, I get `Error: no such option: --file`:

```
$ sudo iotedgedev start --setup --file config/deployment-orig.json 
[sudo] password for sad: 

=======================================
======== ENVIRONMENT VARIABLES ========
=======================================

Environment Variables loaded from: .env (/home/sad/projects/iotedge-dev-test/.env)
Usage: iotedgedev start [OPTIONS]
Try "iotedgedev start -h" for help.

Error: no such option: --file
$ ls -halt config/
total 44K
drwxr-xr-x. 5 sad sad 4.0K Mar 20 06:28 ..
drwxr-xr-x. 2 sad sad 4.0K Feb 15 11:19 .
-rw-rw-r--  1 sad sad 2.9K Jan 26 15:43 .env
-rw-r--r--. 1 sad sad 2.1K Oct 31 13:48 deployment-orig.json
$ iotedgehubdev --version
iotedgehubdev, version 0.6.0
```

What am I doning wrong?

Thank you.

/Henrik

**Comment by blackchoey** [comment]
Hi @HenrikBach1 , could you execute `iotedgedev --version` to check the version of iotedgedev? `--file` option is available starting from v1.1.0. Please execute `pip install --upgrade iotedgedev` if you have a lower version of iotedgedev.

**Comment by HenrikBach1** [comment]
Hi @blackchoey , Yes, that fixed the error. Thank you.
